frontend/src/pages/ActivityReport/index.css

```diff
@@ -1,13 +1,29 @@
-.smart-hub--form-section {
-  margin-top: 2rem;
+.smart-hub--form-group {
+  margin-top: 1.5rem;
 }
 
 .smart-hub--report-checkbox {
   margin-top: 1.25rem;
 }
 
+.smart-hub--form-section {
+  margin-top: 2rem;
+}
+
+.smart-hub--remove-resource {
+  margin-top: 8px !important;
+  margin-left: 4px;
+}
+
+.smart-hub--virtual-delivery-group {
+  border-left-color: #0166ab;
+  border-left-style: solid;
+  border-left-width: 4px;
+  padding-left: 10px;
+}
+
 .smart-hub--report-legend .usa-legend {
-  font-family: Merriweather;
+  font-family: 'Source Sans Pro Web';
```

Comment:
I noticed that `.smart-hub--form-required` has this as `SourceSansPro` which doesn't work in Firefox, so the *Required* labels are rendering as Times.

Also, there are [`.font-ui` and `.font-family-sans` classes](https://designsystem.digital.gov/utilities/font-size-and-family/) that achieve this.